Sources/AppcuesKit/Data/Networking/NetworkClient.swift

```diff
@@ -91,11 +154,43 @@ internal class NetworkClient: Networking {
 
         dataTask.resume()
     }
+
+    // version that does not decode any response object, assumes empty or discards
+    private func handleRequest(_ urlRequest: URLRequest,
```

Comment:
some redundancy here with `handleRequest<T>` but I couldn't think of a great way to combine them that didn't make it more complicated and harder to read 🤷‍♂️ Open to ideas if this is problematic.